examples/seismic/acoustic/accuracy.ipynb

```diff
@@ -50,7 +51,15 @@
    "cell_type": "code",
    "execution_count": 3,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
```

Comment:
Maybe this printing is a bit redundant, at least for `dt, t0, nt`.
So I would suggest to only print `tn`.

Reply:
Personally I like to see the details, but understood if everyone does not. t0 is pretty obvious, agree, how about dt and tn?

Reply:
I like it like that, it's a tuto nothing is too much info, we may know it as dev because we used to it be butter to put mre out than less.